elastio-ec2-backup_ecs_stopped_task/readme.md

```diff
@@ -1,67 +1,112 @@
-        1. Soft limit: `4096`
-        2. Hard limit: `4096`
-    14. Press `Add`.
-    15. Lastly, press `Create` button.
-6. Run the ECS schedule task.
-    1. Select your cluster in ECS Clusters.
-    2. In the navigation tab choose `Scheduled Tasks`.
-    3. Press `Create` button.
-    4. Enter `Schedule rule name`=>`Target id`.
-    5. Launch type choose `Fargate`.
-    6. Task Definition, choose your task definition.
-    7. Select a VPC from the list in the Cluster VPC field.
-    8. Add at least one subnet.
-    9. Enable Auto-assign public IP.
-    10. Press `Create` button. This will run the task according to the selected schedule.
-7. Get logs.
-    1. Open AWS Cloud Watch Service
-    2. In the left side menu, choose `Logs`=>`Log group`.
-    3. In the search field, enter your schedule task definition name.
-    4. Choose in result your task definition with logs.
\ No newline at end of file
+# Elastio EC2 backup ECS stopped task
+
+## Step 1 - Create ECR polices
+
+1. From AWS console, select "IAM service" => `Policies` => "Create Policy".
+1. Under the **Service** tab select `Elastic Container Registry`.
+1. Under the **Actions** tab select `All Elastic Container Registry actions (ecr:*)`.
+1. Under the **Resources** tab select `specific` =>`Add ARN`.
+    Then choose the region, leave your account number and
+    select "Any" for the Repository name.
+1. Enter 'ECR_FullAccess' as a policy name.
+
+## Step 2 - Roles
+
+1. From AWS console, select IAM service =>`Roles`.
+1. If you already have a role for ECS tasks, add the following permissions:
+
+    1. AmazonECSTaskExecutionRolePolicy,
+    1. SecretsManagerReadWrite,
+    1. AmazonECS_FullAccess,
+    1. ECR_FullAccess,
+    1. ElastioBackupAdmin
+
+    or create a new role with these permissions.
+Creating a new role follow these steps:
+
+1. In the navigation pane, choose "Roles" =>"Create role".
+1. In the "Select type of trusted entity section",
+    choose "AWS service" => "Elastic Container Service".
+1. In "Select your use case", select the "Elastic Container Service Task",
+    then choose "Next": "Permissions".
+1. In the "Attach permissions policies" section, search for
+
+    1. AmazonECSTaskExecutionRolePolicy,
+    1. SecretsManagerReadWrite,
+    1. AmazonECS_FullAccess,
+    1. ECR_FullAccess,
+    1. ElastioBackupAdmin
+
+    select "Policies", then choose "Next": "Tags".
+1. For Add tags (optional), specify custom tags to
+    associate with the policy and then choose "Next": "Review".
+1. For Role name, type `ecs_task_execution_role` and choose "Create role".
+
+## Step 3 - Build and push a docker image
+
+> You can find the latest version by following this link.
+<https://gallery.ecr.aws/elastio-dev/elastio-cli>
+
+1. Edit file contrib/elastio-ec2-backup-ecs-stopped-task/failed_ecs_handler_config.py
+    and put data about topic, brokers, vault.
+1. Create ECR repository.
+1. Register your docker cli by getting the ecr public image with the command:
```

Comment:
Once again, abbreviations in English are *always* capitalized.  Please fix this throughout

```suggestion
1. Register your docker CLI by getting the ECR public image with the command:
```